sample/src/main/java/co/nimblehq/sample/AuthService.kt

```diff
@@ -0,0 +1,13 @@
+package co.nimblehq.sample
+
+import co.nimblehq.sample.entity.UserEntity
+import co.nimblehq.survey.sdk.request.LoginRequest
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface AuthService {
+
+    @POST("/api/v1/oauth/token")
+    suspend fun loginEmail(@Body request: LoginRequest): UserEntity
```

Comment:
I guess we will move the logic to handle login to SDK on the next phase 🤔

Reply:
@hoangnguyen92dn Yes, maybe next phase we will support authen service inside the SDK